scripts/init-submodules-no-riscv-tools.sh

```diff
@@ -11,14 +11,14 @@ scripts_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 # ignore riscv-tools for submodule init recursive
 # you must do this globally (otherwise riscv-tools deep
 # in the submodule tree will get pulled anyway
-git config --global submodule.riscv-tools.update none
+git config --global submodule.toolchains/riscv-tools.update none
```

Comment:
this can be handled out of band, but it would be best if we tried not to use  `--global` where possible

Reply:
better for and out-of-band discussion with the git experts

Reply:
Isn't this a pretty simple change / can't you just remove the `--global`?

Reply:
It could be, but then, why was it there to the beginwith? I'm assuming there was some good reason...

Reply:
In the comments:
```
 # you must do this globally (otherwise riscv-tools deep
 # in the submodule tree will get pulled anyway
```
Where is `riscv-tools` in the submodule tree though? Is this vestigial from when rocket-chip included it?

Reply:
FWIW I just tried it without global and it behaved as desired.

Reply:
It is vestigial. At one point there three instances of riscv-tools in FireSim's submodule hierarchy and that was the quick and dirty way to deal with it.

We should avoid using `--global` though since it's exactly that -- global. And it pollutes the user's gitconfig.

Reply:
ok, ok, I changed it